testsuite/forge/src/main.rs

```diff
@@ -54,11 +55,14 @@ fn main() -> Result<()> {
             &args.options,
         )
     } else {
-        forge_main(
-            k8s_test_suite(),
-            K8sFactory::new(args.root_key.clone(), args.treasury_compliance_key.clone()),
-            &args.options,
-        )
+        let k8s_factory;
+        if args.root_key.is_empty() && args.treasury_compliance_key.is_empty() {
+            k8s_factory = K8sFactory::new_internal().unwrap();
+        } else {
+            k8s_factory =
+                K8sFactory::new(args.root_key.clone(), args.treasury_compliance_key.clone());
+        }
```

Comment:
For consistency we probably always want to fetch the keys from the testnet vault instance right?

Reply:
yea, we do not need this else routine I reckon

Reply:
edited